js/app.js

```diff
@@ -44,55 +19,21 @@ window.addEventListener('DOMContentLoaded', function() {
   var messageArea = document.getElementById('message-container');
 
   function enableImport() {
-    var timeEnable;
-    if (!accessToken || !tokenValidity) {
-      timeEnable = -1
-    } else {
-      timeEnable = tokenValidity - Date.now();
-    }
-
-    if (timeEnable > 0) {
-      importButton.style.display = '';
-      authorizeButton.querySelector('div[data-l10n-id]').dataset.l10nId =
-        'reauthorize';
+    importButton.style.display = '';
+    authorizeButton.querySelector('div[data-l10n-id]').dataset.l10nId =
+      'reauthorize';
+    navigator.mozL10n.once(() => {
       navigator.mozL10n.translate(authorizeButton);
-      setTimeout(disableImport, timeEnable);
-    } else {
-      disableImport();
-    }
+    });
   }
 
   function disableImport() {
     importButton.style.display = 'none';
     authorizeButton.querySelector('div[data-l10n-id]').dataset.l10nId =
       'authorize';
-    navigator.mozL10n.translate(authorizeButton);
-  }
-
-  function loadPersistedState() {
-    accessToken = localStorage.getItem(ACCESS_TOKEN_KEY);
-    tokenValidity = new Date(localStorage.getItem(TOKEN_VALIDITY_KEY));
-    navigator.mozL10n.ready(enableImport);
-  }
-
-  function saveAccessToken(parameters) {
-    accessToken = parameters.access_token;
-    localStorage.setItem(ACCESS_TOKEN_KEY, accessToken);
-    tokenValidity = new Date(Date.now() + parameters.expires_in * 1000);
-    localStorage.setItem(TOKEN_VALIDITY_KEY, tokenValidity);
-  }
-
-  function tokenDataReady(e) {
-    var parameters = e.data;
-    if (e.origin !== location.origin) {
-      return;
-    }
-    if (!parameters || !parameters.access_token) {
-      return;
-    }
-
-    saveAccessToken(parameters);
-    enableImport();
+    navigator.mozL10n.once(() => {
+      navigator.mozL10n.translate(authorizeButton);
+    });
```

Comment:
Ok, that's fine with me.

For my education, how do you indicate to l10n.js to translate a fragment you add dynamically to the dom?